internal/server/workflow.go

```diff
@@ -90,6 +90,9 @@ func loggingInfoForWorkflow(client WorkflowInformation, tmpl Template, defaultWo
 	return func(w http.ResponseWriter, r *http.Request) error {
 		ctx := getContext(r)
 		search, _ := strconv.Atoi(r.FormValue("page"))
+		if search < 1 {
+		    search = 1
+		}
```

Comment:
Do we need this for trying to look at a page larger then we have displayed ie 3 pages and they look for page 4?

Reply:
It shouldn't trigger if they're on page 3 and click for page 4 because at that point search = 4